### PR TITLE
Add missing arxiv papers that were descibed by Rohin

### PR DESCRIPTION
**Pull request by mruwnik**
I noticed that these arxiv papers had been summarized by Rohin, but weren't found when merging

### CODE REVIEW

**Review by ccstan99 — DISMISSED** [review]
Am I reading the diff correctly, that you added ~320 arxiv papers referenced in alignment_newsletter.xlsx to the bottom of this .csv file in GH? You might want to put quotes or something around the authors so they get treated as elements of 1 column as opposed to separate columns each. That would help it getting aligned to the correct column. Top 4 rows are existing data, bottom rest are new data format with each author in separate column.
Screenshot 2023-06-19 at 8 09 16 PM

FYI, based on PR #33, it's not clear that this file is/was used as the source of @jknowak's scrape. 

Eventually, this list should be kept updated outside of GH. We'll need to decide on the longterm home but for now, I uploaded a copy of your file to [gdrive](https://drive.google.com/drive/u/0/folders/1QBJPWXa9PcGhSslReLy3xHx1FwUKNr3h) both as a separate [.csv](https://drive.google.com/file/d/11bjwFTArz8xmF-BbwyQNU2z8QpxnwTeN/view?usp=drive_link) file and [tab in Google sheets](https://docs.google.com/spreadsheets/d/1pgG3HzercOhf4gniaqp3tBc3uvZnHpPhXErwHcthmbI/edit#gid=501896828), if we want to view multiple sources from single point. 

Also, this .csv file has a mix of arxiv + other sources. Based on our last discussion about having a master file with special list of documents, blog posts, etc with metadata, I'll probably start from this list then separate the arxiv papers from the special_docs.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The following list of sources may change and items may be renamed:
 - gdrive_ebooks - books include [Superintelligence](https://www.goodreads.com/book/show/20527133-superintelligence), [Human Compatible](https://www.goodreads.com/book/show/44767248-human-compatible), [Life 3.0](https://www.goodreads.com/book/show/34272565-life-3-0), [The Precipice](https://www.goodreads.com/book/show/50485582-the-precipice), and others
 - [generative.ink](https://generative.ink/posts/)
 - [gwern_blog](https://gwern.net/)
-- [import.ai](https://importai.substack.com)
+- [importai](https://importai.substack.com)
```
Comment (ccstan99) [inline README.md:27]: Snuck in a couple small tweaks to README.md

**Review by ccstan99 — APPROVED** [review]

**Comment by mruwnik** [comment]
oops - good point. Yup, these are mentioned in the alignment newsletter, so I thought they should probably go here. I only added the missing arxiv papers, but can also add the non arxiv ones

**Comment by mruwnik** [comment]
I'll close this for now - it should be done properly with the other files

**Pull request closed without merging**